scene/gui/text_edit.cpp

```diff
@@ -4695,6 +4696,67 @@ void TextEdit::paste() {
 		clipboard += ins;
 	}
 
+#ifdef TOOLS_ENABLED
+	// In the (script) editor, optionally convert indentation to configured one
+	if (Engine::get_singleton()->is_editor_hint() && EDITOR_GET("text_editor/indent/convert_indent_on_paste")) {
+		int indent_type = EDITOR_GET("text_editor/indent/type"); // 0 is tabs, 1 is spaces
+		bool has_indent = true;
+		String old_indent;
+		String new_indent;
+		Vector<String> lines = clipboard.split("\n");
+		if (indent_type == 0) { // tabs
+			// We need to convert spaces to tabs, so we have to figure out what indent size is used in the clipboard
+			int min_indent_size = 10000;
```

Comment:
You might use `unsigned int min_indent_size = -1`, to avoid hardcoding the value.
Alternatively, the `if` below might check if `min_indent_size` is -1.

Reply:
> You might use `unsigned int min_indent_size = -1`, to avoid hardcoding the value.

Seems a bit hacky, I guess compilers might even throw a warning. If needed I could use `INT_MAX` though.

> Alternatively, the `if` below might check if `min_indent_size` is -1.

I wouldn't be able to get `count < min_indent_size` to true.

Reply:
> I wouldn't be able to get count < min_indent_size to true.

I mean, checking for `count < min_indent_size || min_indent_size == -1`

`INT_MAX` sounds better though.